sys/aarch64/pl011.c

```diff
@@ -19,16 +21,33 @@
 #define UART0_BASE BCM2835_PERIPHERALS_BUS_TO_PHYS(BCM2835_UART0_BASE)
 #define UART_BUFSIZE 128
 
+static inline void set4(resource_t *r, int o, uint32_t v) {
+  bus_write_4(r, o, bus_read_4(r, o) | v);
+}
+
+static inline void clr4(resource_t *r, int o, uint32_t v) {
+  set4(r, o, bus_read_4(r, o) & ~v);
```

Comment:
Shouldn't this be `bus_write_4(r, o, bus_read_4(r, o) & ~v)`?

Reply:
It should. Fixed.